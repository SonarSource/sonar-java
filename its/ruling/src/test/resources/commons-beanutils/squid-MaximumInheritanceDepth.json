{
'commons-beanutils:commons-beanutils:src/main/java/org/apache/commons/beanutils2/NestedNullException.java':[
27,
],
}
