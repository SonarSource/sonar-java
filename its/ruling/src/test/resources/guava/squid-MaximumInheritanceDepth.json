{
'com.google.guava:guava:src/com/google/common/collect/AbstractMapBasedMultimap.java':[
1005,
1048,
],
'com.google.guava:guava:src/com/google/common/collect/ConcurrentHashMultiset.java':[
557,
],
'com.google.guava:guava:src/com/google/common/collect/ContiguousSet.java':[
42,
],
'com.google.guava:guava:src/com/google/common/collect/DescendingImmutableSortedMultiset.java':[
25,
],
'com.google.guava:guava:src/com/google/common/collect/DescendingImmutableSortedSet.java':[
28,
],
'com.google.guava:guava:src/com/google/common/collect/EmptyContiguousSet.java':[
32,
],
'com.google.guava:guava:src/com/google/common/collect/ForwardingNavigableMap.java':[
331,
],
'com.google.guava:guava:src/com/google/common/collect/ForwardingNavigableSet.java':[
163,
],
'com.google.guava:guava:src/com/google/common/collect/ForwardingSortedMap.java':[
104,
],
'com.google.guava:guava:src/com/google/common/collect/ForwardingSortedMultiset.java':[
68,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableRangeSet.java':[
390,
],
'com.google.guava:guava:src/com/google/common/collect/ImmutableSortedAsList.java':[
32,
],
'com.google.guava:guava:src/com/google/common/collect/Maps.java':[
1033,
2866,
2973,
3658,
],
'com.google.guava:guava:src/com/google/common/collect/RegularContiguousSet.java':[
36,
],
'com.google.guava:guava:src/com/google/common/collect/RegularImmutableSortedMultiset.java':[
33,
],
'com.google.guava:guava:src/com/google/common/collect/RegularImmutableSortedSet.java':[
47,
],
'com.google.guava:guava:src/com/google/common/collect/Sets.java':[
1598,
],
'com.google.guava:guava:src/com/google/common/collect/SortedMultisets.java':[
90,
],
'com.google.guava:guava:src/com/google/common/collect/TreeMultimap.java':[
79,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/CycleDetectingLockFactory.java':[
568,
],
'com.google.guava:guava:src/com/google/common/util/concurrent/Futures.java':[
2100,
],
}
