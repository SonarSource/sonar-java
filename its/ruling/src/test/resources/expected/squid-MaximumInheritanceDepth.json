{
'project:com/sun/corba/se/impl/corba/AnyImpl.java':[
68,
76,
],
'project:com/sun/corba/se/impl/dynamicany/DynValueBoxImpl.java':[
42,
],
'project:com/sun/corba/se/impl/dynamicany/DynValueImpl.java':[
42,
],
'project:com/sun/corba/se/impl/encoding/CDROutputObject.java':[
64,
],
'project:com/sun/corba/se/impl/encoding/TypeCodeInputStream.java':[
67,
],
'project:com/sun/corba/se/impl/encoding/TypeCodeOutputStream.java':[
64,
],
'project:com/sun/corba/se/impl/ior/iiop/IIOPProfileTemplateImpl.java':[
61,
],
'project:com/sun/corba/se/internal/Interceptors/PIORB.java':[
32,
],
'project:com/sun/corba/se/internal/POA/POAORB.java':[
31,
],
'project:com/sun/corba/se/internal/corba/ORBSingleton.java':[
32,
],
'project:com/sun/corba/se/internal/iiop/ORB.java':[
31,
],
'project:com/sun/java/swing/plaf/gtk/GTKColorChooserPanel.java':[
41,
],
'project:com/sun/java/swing/plaf/gtk/GTKFileChooserUI.java':[
676,
1012,
1025,
1288,
],
'project:com/sun/java/swing/plaf/motif/MotifCheckBoxMenuItemUI.java':[
45,
],
'project:com/sun/java/swing/plaf/motif/MotifCheckBoxUI.java':[
48,
],
'project:com/sun/java/swing/plaf/motif/MotifComboBoxUI.java':[
96,
],
'project:com/sun/java/swing/plaf/motif/MotifDesktopIconUI.java':[
294,
],
'project:com/sun/java/swing/plaf/motif/MotifFileChooserUI.java':[
306,
416,
439,
453,
467,
481,
639,
650,
758,
],
'project:com/sun/java/swing/plaf/motif/MotifInternalFrameTitlePane.java':[
227,
271,
283,
297,
312,
],
'project:com/sun/java/swing/plaf/motif/MotifMenuUI.java':[
45,
],
'project:com/sun/java/swing/plaf/motif/MotifPasswordFieldUI.java':[
44,
],
'project:com/sun/java/swing/plaf/motif/MotifRadioButtonMenuItemUI.java':[
51,
],
'project:com/sun/java/swing/plaf/motif/MotifRadioButtonUI.java':[
50,
],
'project:com/sun/java/swing/plaf/motif/MotifScrollBarButton.java':[
46,
],
'project:com/sun/java/swing/plaf/motif/MotifTextPaneUI.java':[
44,
],
'project:com/sun/java/swing/plaf/motif/MotifTreeCellRenderer.java':[
49,
],
'project:com/sun/java/swing/plaf/windows/WindowsCheckBoxMenuItemUI.java':[
47,
],
'project:com/sun/java/swing/plaf/windows/WindowsCheckBoxUI.java':[
48,
],
'project:com/sun/java/swing/plaf/windows/WindowsComboBoxUI.java':[
410,
462,
512,
],
'project:com/sun/java/swing/plaf/windows/WindowsFileChooserUI.java':[
420,
459,
476,
596,
978,
1167,
],
'project:com/sun/java/swing/plaf/windows/WindowsMenuUI.java':[
49,
],
'project:com/sun/java/swing/plaf/windows/WindowsPasswordFieldUI.java':[
45,
],
'project:com/sun/java/swing/plaf/windows/WindowsRadioButtonMenuItemUI.java':[
46,
],
'project:com/sun/java/swing/plaf/windows/WindowsRadioButtonUI.java':[
47,
],
'project:com/sun/java/swing/plaf/windows/WindowsScrollBarUI.java':[
276,
],
'project:com/sun/java/swing/plaf/windows/WindowsTableHeaderUI.java':[
74,
],
'project:com/sun/java/swing/plaf/windows/WindowsTextPaneUI.java':[
44,
],
'project:com/sun/java/swing/plaf/windows/WindowsTreeUI.java':[
193,
],
'project:com/sun/java/swing/plaf/windows/XPStyle.java':[
677,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/AncestorPattern.java':[
47,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/IdPattern.java':[
31,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/KeyPattern.java':[
30,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/ParentPattern.java':[
42,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/ProcessingInstructionPattern.java':[
46,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/StepPattern.java':[
65,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/SAXImpl.java':[
80,
],
'project:com/sun/org/apache/xerces/internal/dom/DeferredCDATASectionImpl.java':[
49,
],
'project:com/sun/org/apache/xerces/internal/dom/DeferredDocumentImpl.java':[
46,
],
'project:com/sun/org/apache/xerces/internal/dom/DeferredElementNSImpl.java':[
44,
],
'project:com/sun/org/apache/xerces/internal/dom/PSVIDocumentImpl.java':[
40,
],
'project:com/sun/org/apache/xerces/internal/dom/PSVIElementNSImpl.java':[
40,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMDefaultBaseTraversers.java':[
1470,
1556,
],
'project:com/sun/org/apache/xpath/internal/axes/AttributeIterator.java':[
34,
],
'project:com/sun/org/apache/xpath/internal/axes/ChildTestIterator.java':[
37,
],
'project:com/sun/org/apache/xpath/internal/axes/FilterExprIterator.java':[
31,
],
'project:com/sun/org/apache/xpath/internal/axes/OneStepIterator.java':[
39,
],
'project:com/sun/org/apache/xpath/internal/axes/OneStepIteratorForward.java':[
38,
],
'project:com/sun/org/apache/xpath/internal/axes/UnionChildIterator.java':[
35,
],
'project:com/sun/org/apache/xpath/internal/axes/WalkingIteratorSorted.java':[
33,
],
'project:com/sun/org/apache/xpath/internal/functions/FuncConcat.java':[
34,
],
'project:com/sun/org/apache/xpath/internal/functions/FuncSubstring.java':[
36,
],
'project:com/sun/org/apache/xpath/internal/functions/FuncTranslate.java':[
33,
],
'project:com/sun/org/apache/xpath/internal/functions/FunctionMultiArgs.java':[
36,
],
'project:java/awt/event/MouseWheelEvent.java':[
100,
],
'project:java/nio/channels/ClosedByInterruptException.java':[
42,
],
'project:java/rmi/server/SocketSecurityException.java':[
34,
],
'project:java/util/DuplicateFormatFlagsException.java':[
38,
],
'project:java/util/FormatFlagsConversionMismatchException.java':[
37,
],
'project:java/util/IllegalFormatCodePointException.java':[
39,
],
'project:java/util/IllegalFormatConversionException.java':[
38,
],
'project:java/util/IllegalFormatFlagsException.java':[
37,
],
'project:java/util/IllegalFormatPrecisionException.java':[
35,
],
'project:java/util/IllegalFormatWidthException.java':[
34,
],
'project:java/util/MissingFormatArgumentException.java':[
39,
],
'project:java/util/MissingFormatWidthException.java':[
37,
],
'project:java/util/UnknownFormatConversionException.java':[
37,
],
'project:java/util/UnknownFormatFlagsException.java':[
37,
],
'project:javax/security/auth/login/AccountExpiredException.java':[
39,
],
'project:javax/security/auth/login/AccountLockedException.java':[
37,
],
'project:javax/security/auth/login/AccountNotFoundException.java':[
36,
],
'project:javax/security/auth/login/CredentialExpiredException.java':[
41,
],
'project:javax/security/auth/login/CredentialNotFoundException.java':[
36,
],
'project:javax/swing/DefaultListCellRenderer.java':[
344,
],
'project:javax/swing/JApplet.java':[
564,
],
'project:javax/swing/JButton.java':[
310,
],
'project:javax/swing/JCheckBox.java':[
78,
337,
],
'project:javax/swing/JCheckBoxMenuItem.java':[
91,
296,
],
'project:javax/swing/JColorChooser.java':[
607,
],
'project:javax/swing/JDialog.java':[
1248,
],
'project:javax/swing/JEditorPane.java':[
1643,
1697,
1777,
2123,
],
'project:javax/swing/JFormattedTextField.java':[
180,
],
'project:javax/swing/JFrame.java':[
898,
],
'project:javax/swing/JMenu.java':[
112,
646,
1397,
],
'project:javax/swing/JMenuItem.java':[
832,
],
'project:javax/swing/JPasswordField.java':[
77,
406,
],
'project:javax/swing/JPopupMenu.java':[
436,
],
'project:javax/swing/JRadioButton.java':[
89,
287,
],
'project:javax/swing/JRadioButtonMenuItem.java':[
86,
269,
],
'project:javax/swing/JSpinner.java':[
944,
1122,
1264,
],
'project:javax/swing/JTable.java':[
5352,
5359,
5371,
5383,
5392,
],
'project:javax/swing/JTextArea.java':[
790,
],
'project:javax/swing/JTextField.java':[
946,
],
'project:javax/swing/JTextPane.java':[
82,
],
'project:javax/swing/JToggleButton.java':[
387,
],
'project:javax/swing/JToolBar.java':[
533,
],
'project:javax/swing/colorchooser/ColorChooserPanel.java':[
41,
],
'project:javax/swing/colorchooser/DefaultSwatchChooserPanel.java':[
54,
320,
347,
],
'project:javax/swing/event/MenuDragMouseEvent.java':[
50,
],
'project:javax/swing/event/MenuKeyEvent.java':[
50,
],
'project:javax/swing/plaf/basic/BasicArrowButton.java':[
48,
],
'project:javax/swing/plaf/basic/BasicCheckBoxUI.java':[
52,
],
'project:javax/swing/plaf/basic/BasicComboBoxEditor.java':[
129,
],
'project:javax/swing/plaf/basic/BasicComboBoxRenderer.java':[
142,
],
'project:javax/swing/plaf/basic/BasicInternalFrameTitlePane.java':[
771,
],
'project:javax/swing/plaf/basic/BasicOptionPaneUI.java':[
1341,
1434,
],
'project:javax/swing/plaf/basic/BasicSplitPaneDivider.java':[
388,
444,
],
'project:javax/swing/plaf/basic/BasicTabbedPaneUI.java':[
3494,
],
'project:javax/swing/plaf/basic/BasicTextAreaUI.java':[
230,
],
'project:javax/swing/plaf/basic/BasicTextFieldUI.java':[
169,
],
'project:javax/swing/plaf/basic/BasicToolBarUI.java':[
474,
508,
],
'project:javax/swing/plaf/metal/MetalCheckBoxUI.java':[
54,
],
'project:javax/swing/plaf/metal/MetalComboBoxButton.java':[
51,
],
'project:javax/swing/plaf/metal/MetalComboBoxEditor.java':[
55,
],
'project:javax/swing/plaf/metal/MetalComboBoxUI.java':[
367,
],
'project:javax/swing/plaf/metal/MetalFileChooserUI.java':[
383,
533,
852,
1041,
],
'project:javax/swing/plaf/metal/MetalRadioButtonUI.java':[
56,
],
'project:javax/swing/plaf/metal/MetalScrollButton.java':[
53,
],
'project:javax/swing/plaf/metal/MetalSplitPaneDivider.java':[
105,
214,
],
'project:javax/swing/plaf/synth/SynthArrowButton.java':[
36,
],
'project:javax/swing/plaf/synth/SynthCheckBoxMenuItemUI.java':[
44,
],
'project:javax/swing/plaf/synth/SynthCheckBoxUI.java':[
40,
],
'project:javax/swing/plaf/synth/SynthComboPopup.java':[
43,
],
'project:javax/swing/plaf/synth/SynthDesktopIconUI.java':[
63,
],
'project:javax/swing/plaf/synth/SynthFormattedTextFieldUI.java':[
37,
],
'project:javax/swing/plaf/synth/SynthListUI.java':[
200,
],
'project:javax/swing/plaf/synth/SynthMenuUI.java':[
43,
],
'project:javax/swing/plaf/synth/SynthPasswordFieldUI.java':[
40,
],
'project:javax/swing/plaf/synth/SynthRadioButtonMenuItemUI.java':[
40,
],
'project:javax/swing/plaf/synth/SynthRadioButtonUI.java':[
39,
],
'project:javax/swing/plaf/synth/SynthScrollBarUI.java':[
377,
406,
],
'project:javax/swing/plaf/synth/SynthTabbedPaneUI.java':[
925,
],
'project:javax/swing/plaf/synth/SynthTableHeaderUI.java':[
219,
],
'project:javax/swing/plaf/synth/SynthTableUI.java':[
709,
747,
],
'project:javax/swing/plaf/synth/SynthTextPaneUI.java':[
50,
],
'project:javax/swing/plaf/synth/SynthTreeUI.java':[
686,
763,
],
'project:javax/swing/table/DefaultTableCellRenderer.java':[
394,
],
'project:javax/swing/table/TableColumn.java':[
804,
],
'project:javax/swing/text/html/FrameView.java':[
461,
],
'project:javax/swing/text/html/HTMLEditorKit.java':[
1827,
],
'project:javax/swing/text/html/LineView.java':[
42,
],
'project:javax/swing/text/html/ParagraphView.java':[
46,
],
'project:javax/swing/tree/DefaultTreeCellEditor.java':[
588,
],
'project:javax/xml/ws/http/HTTPException.java':[
37,
],
'project:javax/xml/ws/soap/SOAPFaultException.java':[
53,
],
}
